src/test/java/spoon/test/UnresolvedBugTest.java

```diff
@@ -12,31 +12,52 @@
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.net.URL;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 
 import com.google.gson.Gson;
 import com.google.gson.JsonObject;
 
+import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Test;
-import org.junit.experimental.categories.Category;
 
+import org.junit.runner.Description;
+import org.junit.runner.JUnitCore;
+import org.junit.runner.notification.Failure;
+import org.junit.runner.notification.RunListener;
+import org.junit.runner.notification.RunNotifier;
+import org.junit.runners.BlockJUnit4ClassRunner;
+import org.junit.runners.model.FrameworkMethod;
+import org.junit.runners.model.InitializationError;
 import spoon.FluentLauncher;
 import spoon.reflect.declaration.CtMethod;
 import spoon.reflect.visitor.filter.TypeFilter;
 
 public class UnresolvedBugTest {
 
 	private static final String OPEN_ISSUE_TEXT = "open";
-	private List<CtMethod<?>> testMethods = findTestMethods();
+	private static List<CtMethod<?>> testMethods = findTestMethods();
 	private final String githubURL = "https://api.github.com/repos/INRIA/spoon/issues/";
 
+
+	@BeforeClass
+	public static void setup() {
+		testMethods = testMethods.stream()
+				.filter(v -> v.hasAnnotation(Test.class) &&  v.hasAnnotation(Ignore.class)
+						&& v.getAnnotation(Ignore.class).value().equals("UnresolvedBug"))
```

Comment:
how about https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#equalsIgnoreCase(java.lang.String) instead of `equals`?

Reply:
Sure! Why not!